inst/app.crunch.io/api/datasets/3.json

```diff
@@ -20,7 +20,8 @@
     },
     "fragments": {
         "table": "https://app.crunch.io/api/datasets/3/table/",
-        "preferences": "https://app.crunch.io/api/datasets/3/preferences/"
+        "preferences": "https://app.crunch.io/api/datasets/3/preferences/",
+        "pk": "https://app.crunch.io/api/datasets/3/pk/"
```

Comment:
Shouldn't this mockfile below be a shoji:entity with `"pk":[]`? Check what the API actually returns.